src/ligneCompleteEtCalculPoints.cpp

```diff
@@ -0,0 +1,95 @@
+//#include <board.h>
+#include <cstdio>
+#include <iostream>
+//#include <algorithme>
+//#include <SFML/Graphics.hpp>
+
+using namespace sf;
+using namespace std;
+using std::cout; 
+using std::vector;
+using std::array; 
+using std::rotate;
+using std::endl;
+
+
+int incrementeScore(int score){
+    
+    score = score + 100; // pour chaque ligne supprimé, on ajout 100 points
+    return score;
+}
+
+void decaleLigne(int a, bool*verif){
+    
+    int i=0;
+    int j,k;
+    int b=0;
+    int Tab[a] = {false};
+        
+    for(j=0;j<20;j++){
+        if (verif[j]==true){ //On détermine quel lignes ont été supprimés 
+            Tab[i]=j; // et on les stocke leur numéros dans un tableau
+            i = i + 1;
+        }
+    }    
+            
+    for (i=0;i<a;i++){ //Décalle tous le tableau de 1 case vers le bas pour les lignes > à la ligne qui est supprimé 
+        if(i==a-1){ //Si on arrive en haut du tableau :
+            for(j=Tab[i];j<20-a;j++){
+                for(k=0;k<10;k++){
+                    board[i][j] = board[j+1+b][k];
```

Comment:
Variable board : either is a private variable or comes from a specific object.
Is out of scope anyway right now in your code.